docusaurus/video/docusaurus/docs/api/basics/get_started.mdx

```diff
@@ -36,15 +36,50 @@ go get github.com/GetStream/video-go-sdk
 </TabItem>
 </Tabs>
 
+## Creating client
+
+<Tabs groupId="examples">
+<TabItem value="js" label="JavaScript">
+
+```js
+const client = new StreamVideoClient("<Your API key>", {
+  secret: "<API secret>",
+});
+```
+
+</TabItem>
+<TabItem value="py" label="Python">
+
+```py
+TODO
+```
+
+</TabItem>
+<TabItem value="go" label="Golang">
+
+```go
+TODO
+```
+
+</TabItem>
+</Tabs>
+
 ## Creating user tokens
 
 <Tabs groupId="examples">
 <TabItem value="js" label="JavaScript">
 
 ```js
-function helloWorld() {
-    console.log('Hello, world!');
-}
+const userId = "john";
+// exp and iat are optional
+// the token will be valid for 1hour
+const exp = Math.round(new Date().getTime() / 1000) + 60 * 60;
+const iat = Math.round(new Date().getTime() / 1000);
+client.createToken(userId, exp, iat);
+
+// for anonymous tokens
```

Comment:
thats a bit confusing to say anonymous tokens, these are user tokens that allow userId to join call1 and call2